README.md

```diff
@@ -94,110 +121,164 @@ $ grunt vendor
 $ grunt
 ```
 
+### 2. Run Elasticsearch
+
+To launch Elasticsearch, first find out where your Elasticsearch config file is located.
+You can do this with [Homebrew](http://brew.sh) using:
+
+```bash
+$ brew info elasticsearch
+```
+
+The last line of that output should be the command you need to launch Elasticsearch
+with the proper path to its configuration file. For example, it may look like:
+
+```
+$ elasticsearch --config=/Users/<YOUR MAC OSX USERNAME>/homebrew/opt/elasticsearch/config/elasticsearch.yml
+```
+
```

Comment:
I'm currently using `alias elsup="elasticsearch --config=/Users/[USER]/homebrew/opt/elasticsearch/config/elasticsearch.yml"` in my bash profile